framework/src/executioners/Transient.C

```diff
@@ -476,7 +476,7 @@ Transient::endStep(Real input_time)
 
   _last_solve_converged = lastSolveConverged();
 
-  if (_last_solve_converged && !_xfem_repeat_step)
+  if (_last_solve_converged && !_xfem_repeat_step && _time_stepper->enableOutput())
```

Comment:
should it include:
`    //output`
   ` if (_time_interval && (_time + _timestep_tolerance >= _next_interval_output_time))`
      `_next_interval_output_time += _time_interval_output_interval; `

within the if statement?

I'm not sure what this does.